bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -180,6 +186,13 @@ struct BlendedBlockHash {
   uint8_t SuccHash{0};
 };
 
+/// A data object containing function matching information.
+struct FunctionMatchingData {
+public:
```

Comment:
nit: public is not needed for struct since it's the default. also if we only have one counter, the struct seems like an overkill. we could just keep it simple and use an integer counter for now without a struct, and expand later if needed.